semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/Steps.scala

```diff
@@ -50,6 +50,11 @@ class Steps[A](val traversal: Traversal[A]) extends AnyVal {
   def p(implicit show: Show[A] = Show.default): List[String] =
     traversal.toList.map(show.apply)
 
+  @Doc(info = "execute this traversal and show the pretty-printed results in `less`")
+  // uses scala-repl-pp's `#|^` operator which let's `less` inherit stdin and stdout
+  def browse(implicit show: Show[A] = Show.default): Unit =
+    traversal.map(show.apply) #|^ "less"
```

Comment:
I think this will break on Windows :(

Reply:
On Windows we have `more` available in the standard installation.

Reply:
It will, but this never worked on windows...
For reference, here's how ammonite did it: https://github.com/com-lihaoyi/Ammonite/blob/main/amm/repl/api/src/main/scala/ammonite/runtime/tools/Tools.scala#L218

That's not to say that it'd be nice if it did work on windows. I tried with `more` which apparently is available on most windows setups, but somehow the jvm doesn't find the command. Will give it another try later - if you have any additional ideas/thoughts, let me know